### DISCUSSION TITLE
How to solve false positives

### DISCUSSION THREAD

**Discussion by YukiInu** [JetBrains/Qodana#186] [Q&A] [answered]
I have the possibilities to exclude a file, an inspection or a category, but I just want to exclude one instance of a problem.

In this case it is Deprecation/Code maturity but I have an annotation by a third party that will stay for backwards compatibility (Mongock ) and will never be removed.

Is there or will there be a way to solve this with Qodana.
I'm not sure if Quodana is really useful or a Sonarqube alternative without this.

**Comment by brichbash** [comment, marked as answer]
Hello,

At the moment, you cannot exclude an instance of a problem, but you can exclude the inspection only for a method/class by adding the `@SuppressWarnings("<inspection-id>")` annotation. In your case, if you want to exclude the "Deprecated API usage" inspection, adding `@SuppressWarnings("deprecation")` to class/method should ignore the inspection.
